package-lock.json

```diff
@@ -1,5 +1,13 @@
 {
-  "name": "@hmpps-book-secure-move-frameworks/2.4.0",
-  "version": "2.5.1",
-  "lockfileVersion": 1
+  "name": "@hmpps-book-secure-move-frameworks/2.5.2",
+  "version": "2.5.2",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {
+    "": {
```

Comment:
Is this right? How come it has "" as a key? Far be it from me to criticise npm for how it generates a lockfile...

Reply:
oh apparently that's notmal for the root project
https://docs.npmjs.com/cli/v10/configuring-npm/package-lock-json